### PR TITLE
Update docs

### PR DESCRIPTION
**Pull request by underground20**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ The CustomIndexBundle allows create index for doctrine entities using annotation
```
Comment (muxx) [inline README.md:4]: > annotation/attribute

only attribute

File: README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ The CustomIndexBundle allows create index for doctrine entities using annotation
 
 ## Installation
 
-CustomIndexBundle requires Symfony 5 or higher. Work only with PostgreSQL.
+CustomIndexBundle requires Symfony 5 or higher. Works only with PostgreSQL.
 
 Run into your project directory:
 ```
```
Comment (muxx) [inline README.md:10]: Code below is outdated. I thinks we should upgrade it to php8 (return type, short array syntax)

File: README.md [hunk 3]
```diff
@@ -100,77 +91,19 @@ You may use `dump-sql` parameter for dump sql with `DROP/CRATE INDEX` commands
 php app/console intaro:doctrine:index:update --dump-sql
```
Comment (muxx) [inline README.md:86]: > DROP/CRATE INDEX

DROP/CREATE INDEX

File: README.md [hunk 4]
```diff
@@ -100,77 +91,19 @@ You may use `dump-sql` parameter for dump sql with `DROP/CRATE INDEX` commands
...
 ```
 
-## Some annotation examples
+## Examples
 
 Create index using `pg_trgm` extension:
 ```
```
Comment (muxx) [inline README.md]: add php type of code to highlight attribute

**Review by muxx — COMMENTED** [review]
Check the same comment in UPGRADE file, ok

File: README.md (hunk 4 above)
Comment (underground20) [reply README.md]: added

File: README.md (hunk 3 above)
Comment (underground20) [reply README.md:86]: fixed

File: README.md (hunk 1 above)
Comment (underground20) [reply README.md:4]: fixed

File: README.md (hunk 2 above)
Comment (underground20) [reply README.md:10]: updated

**Pull request merged**